xml/art_admin_slemicro_cockpit.xml

```diff
@@ -125,945 +140,2095 @@
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Bridge</term>
+      <listitem>
+       <para>
+        by clicking <guimenu>edit</guimenu>, you can edit all parametes of the
+        bridge.
+       </para>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+   </section>
+   <section xml:id="sec-managing-vlan">
+    <title>Managing VLANs</title>
+    <para>
+     A virtual local area network is a logical subnetwork that groups devices
+     from different physical LANs.
+    </para>
+    <para>
+     To create a VLAN, click <guimenu>Add VLAN</guimenu> and select the parent
+     interface, assign an ID number and provide a name for the VLAN.
+    </para>
+    <figure xml:id="fig-adding-vlan">
+     <title>Adding VLAN</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata  fileref="cockpit_adding_vlan.png" width="100%"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="cockpit_adding_vlan.png" width="100%"/>
+      </imageobject>
+      <textobject role="description"><phrase>Cockpit adding VLAN</phrase>
+      </textobject>
+     </mediaobject>
+    </figure>
+    <para>
+     To modify or delete a VLAN, click the VLAN name.
+    </para>
+    <figure xml:id="fig-modifying-vlan">
+     <title>Adding VLAN</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata  fileref="cockpit_modifying_vlan.png" width="100%"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="cockpit_modifying_vlan.png" width="100%"/>
+      </imageobject>
+      <textobject role="description"><phrase>Cockpit adding VLAN</phrase>
+      </textobject>
+     </mediaobject>
+    </figure>
+   </section>
+   <!-- Currently not available in 5.3
+    <section xml:id="sec-networking-management">
+    <title>Managing interfaces</title>
+    <para>
+    By default, the interfaces are started on boot and use DHCP to assign an IP address. However, you can configure an interface to start on other occassions and/or use a static IP address.
+    </para>
+    <para>
+    The <guimenu>Interfaces</guimenu> view also enables you to delete an interface or disable it.
+    </para>
+    </section>
+    </section>
+    <section xml:id="sec-networking-routing">
+    <title>Routing</title>
+    <para>
+    TBD
+    </para>
+    </section>
+    <section xml:id="sec-networking-dns">
+    <title>DNS</title>
+    <para>
+    TBD
+    </para>
+    </section>
+    -->
+  </section>
+ </section>
+ <section xml:id="sec-cockpit-containers">
+  <title>Working with containers</title>
+  <para>
+   After the first login to Cockpit, you need to start Podman. Keep the default
+   check box selected to start Podman automatically on each boot.
+  </para>
+  <para>
+   The <guimenu>Podman containers</guimenu> page enables you to pull images
+   from registries and manage your container. You can also filter the view by
```

Comment:
(...)manage your containers.

Reply:
Was this missed or did you decide to keep it as is? Usually there are more than one container running on given host.